docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
       - ./app:/app
       - /app/frontend/node_modules/
   backend:
-    image: python:3.6
+    image: garywongbc/python-36-gdal:v1
```

Comment:
We shouldn't be accepting images from personal repos.  Please include any builds in the GWells repo.

Reply:
okay.. Ill do that now.. are we okay with the other personal repo?
https://github.com/bcgov/gwells/blob/87b6ed7920ada6858266d68f1d74aadbc0f061e0/docker-compose.yml#L27

Reply:
That's also bad!  Please use a build config to keep everything within our project.  I'll throw the other into the Trello board, since it's an issue.

Reply:
Can you replace `image: garywongbc/python-36-gdal:v1` with a `build:` section?  That way it will build off the Dockerfile in the repo instead of pulling from dockerhub.

```
build:
  context: .
  dockerfile: ./docker-images/python-gdal/Dockerfile
```

Reply:
Fixed the reference to garywongbc Dockerhub with the 'build' section.

Reply:
@DerekRoberts  please doublecheck this fix (now calls build from the Dockerfile that is now in the repo).